packages/protocol/contracts/identity/IdentityProxy.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.13;
+
+import "../common/ExternalCall.sol";
+
+contract IdentityProxy {
+  address public deployer;
+
+  constructor() public {
+    deployer = msg.sender;
+  }
+
+  modifier onlyDeployer() {
+    require(msg.sender == deployer, "Only callable by original deployer");
+    _;
+  }
+
+  function makeCall(address destination, uint256 value, bytes calldata data) external onlyDeployer {
+    ExternalCall.execute(destination, value, data);
```

Comment:
If the identity logic were executed here instead of in the deployer, it would be possible to deploy these in a trust-less standalone fashion, or via the factory. It would also remove an external call from the transaction execution, which is a nice gas savings.